### ISSUE TITLE
Lot of duplicated classes

### ISSUE THREAD

**Issue by Linuus** [ncri/nested_form_fields#29]
Hi!

When I use this gem, for each nested form I add it gets more and more classes appended.

```
Example:
first item: class="nested_fields nested_mailing_mailing_items"
second item: class="nested_fields nested_mailing_mailing_items nested_fields nested_mailing_mailing_items"
third item: class="nested_fields nested_mailing_mailing_items nested_fields nested_mailing_mailing_items nested_fields nested_mailing_mailing_items"

```
and so on...

Is this the intended behavior? It seems weird to duplicate the classes for each item added.

**Comment by ncri** [comment]
Hi, thanks for the hint, seems like a bug.

**Comment by ncri** [comment]
Fixed in 0.6.2
